### ISSUE TITLE
Mouse interaction does not work with ZoomText 2020

### ISSUE THREAD

**Issue by PurpleGranite** [microsoft/terminal#4889]
# Environment
```none
Windows build number: 10.0.18363.0
Windows Terminal version (if applicable): 0.9.433.0

Any other software?
ZoomText 2020
```

# Steps to reproduce
1. Launch ZoomText 2020 (typically at system startup)
2. Launch Windows Terminal
3. Observe:
 - Clicking on the '+' for a new tab does not work
 - Clicking the dropdown for the options menu does not work
 - Clicking and dragging to select text does not work
4. Press [CapsLock]+[Control]+[Enter] to suspend the Zoom functionality of ZoomText
5. Observe that all features work as expected
6. Re-enable ZoomText's Zoom functionality
7. Observe that features as described in step 3 no longer work

# Expected behavior
Windows Terminal's functions should all work correctly when using assistive software such as ZoomText.

# Actual behavior
Windows Terminal's functions do not work correctly - none of the buttons on the 'titlebar' work (including system buttons such as maximise, minimise or close), the new tab button does not work, the menu drop-down does not work, clicking and dragging to select text does not work.

**Comment by DHowett-MSFT** [comment]
I'm willing to bet this is a platform issue. Mouse interactions over the "Xaml island" seem to be broken.

**Comment by DHowett-MSFT** [comment]
Moved to https://github.com/microsoft/microsoft-ui-xaml/issues/2178 after verifying in another application.
